saveVersion.js

```diff
@@ -0,0 +1,3 @@
+var spawn = require("child_process").spawn;
+var git = spawn("git" ["rev-parse", "HEAD"]);
```

Comment:
`const`

Reply:
also, what the fuck? did you never test this lmao. There's a `,` missing between `"git"` and the arguments

Reply:
It's also javascript (js file)

Why was this not incorporated into the class?